spec/components/playbacktogglebutton.spec.ts

```diff
@@ -0,0 +1,50 @@
+import { MockHelper, TestingPlayerAPI } from '../helper/MockHelper';
+import { UIInstanceManager } from '../../src/ts/uimanager';
+import { ToggleButton } from '../../src/ts/components/togglebutton';
+import { ToggleButtonConfig } from '../../src/ts/components/togglebutton';
+import { DOM } from '../../src/ts/dom';
+
+let playerMock: TestingPlayerAPI;
+let uiInstanceManagerMock: UIInstanceManager;
+let mockDomElement: jest.Mocked<DOM>;
+
+let toggleButton: ToggleButton<ToggleButtonConfig>;
+
+describe('ToggleButton', () => {
+  beforeEach(() => {
+    playerMock = MockHelper.getPlayerMock();
+    uiInstanceManagerMock = MockHelper.getUiInstanceManagerMock();
+
+    // Setup DOM Mock
+    mockDomElement = MockHelper.generateDOMMock();
+    jest.spyOn(ToggleButton.prototype, 'getDomElement').mockReturnValue(mockDomElement);
+  });
+
+  describe('aria label', () => {
+    it('should set on label when toggle state is on', () => {
+      const onLocalizer = () => 'on';
```

Comment:
Can we add a second test using a string instead of the function? 🤔 To also cover the non-localized use-case.

Reply:
Done in b1d4f036